spec/dmv_facility_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+RSpec.describe DmvFacility do
+  
+  describe "#exists" do
+    it "exists" do
+      
+      add_facility1 = DmvDataService.new.or_dmv_office_locations
+      # require 'pry'; binding.pry
+      expect(add_facility1).to be_a(Array)
+      expect(add_facility1[0][:title]).to eq("Albany DMV Office")
+        expect(add_facility1[1][:title]).to eq("Ashland DMV Office")
+        # expect(add_facility1.create_facilities.access_name).to eq("Albany DMV Office")
```

Comment:
You need to initialize your DmvFacility object somewhere so you can call `.create_facilities`. Here you are trying to call it on the data that we pass into `create_facilities` but you need to call it on the DmvFacility object. add_facility1 is the collection of data we're passing in as an argument to that method